R/apply_delay.R

```diff
@@ -0,0 +1,81 @@
+#' Apply the delay to generate a point nowcast
+#' @description
+#' This function takes as an input the reporting triangle that we want to
+#' complete with a nowcast and a delay pmf and generates a point estimate of a
+#' completed reporting square (or rectangle). This code is based on the code
```

Comment:
```suggestion
#' Generate a point estimate of a completed reporting square (or rectangle)
#' from a reporting triangle that we want to complete with a nowcast and a 
#' delay pmf. This code is based on the code
```
(always good to start with what the function does)